index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Colin's Website</title>
+	</head>
+	<body>
+		<nav>
+			<a class="page-link" href="#info">About Me</a>
+			<a class="page-link" href="#portfolio">Portfolio</a>
+			<a class="page-link" href="#contact">Contact</a>
+		</nav>
+		<div class="about-me">
+			<h1>Welcome to My Page!</h1>
+			<section class="me">	
+				<img class="personal-photo" alt="Impromptu selfie taken in class" src="IMG_0368.jpg">
+				<p>Greetings, my name is Colin Meier and this webpage will be used to showcase the work I have completed throughout my time at iO Academy.</p>
+			</section>
+			<section id="info">	
+				<h2>About Me</h2>
+				<p>I was born in New York City and have spent much of my career up until now following my passion in wildlife conservation and eco-tourism. My favorite animal is the clouded leopard, pictured below, and I hope to one day travel to Southeast Asia and try to see one in the wild. In addition to my love for animals and ecology I am also a rabid team sports fan, supporting most New York based clubs in addition to Borussia Dortmund for European football.</p>
+			</section>
+			<section class="about-me-photo-container">
```

Comment:
it is a LITTLE confusing because now it seems like it will only contain one photo. sorry!!!!!!! but please add an s